modules/test/playwright/pages/accessibility-menu-web/AccessibilityMenuPage.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page, expect} from '@playwright/test';
+
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+
+export class AccessibilityMenuPage {
+	readonly closeButton: Locator;
+	readonly enableCheckbox: Locator;
+	readonly menuButton: Locator;
+	readonly page: Page;
+	readonly switchButton: Locator;
+	readonly updateButton: Locator;
+
+	constructor(page: Page) {
+		this.closeButton = page.getByLabel('close', {exact: true});
+		this.enableCheckbox = page.getByLabel('Enable Accessibility Menu');
+		this.menuButton = page.getByRole('button', {
+			name: 'Open Accessibility Menu',
+		});
+		this.page = page;
+		this.switchButton = page.getByLabel('Underlined Links');
+		this.updateButton = page.getByRole('button', {name: 'Update'});
+	}
+
+	async checkUnderlinedLinks() {
+		expect(
+			this.page.getByRole('switch', {name: 'Underlined Links'})
+		).not.toBeChecked();
+
+		await this.switchButton.check();
+
+		expect(this.switchButton).toBeChecked();
+
+		await this.closeButton.click();
+	}
+
+	async displayAccessibilityMenu() {
+		await this.page.keyboard.press('Tab');
+		await this.page.keyboard.press('Tab');
+
+		expect(this.menuButton).toBeVisible();
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: this.page.locator('.modal-dialog'),
+			trigger: this.menuButton,
+		});
+	}
+
+	async enableAccessibilityMenu() {
+		await this.enableCheckbox.check();
```

Comment:
This function should probably have an assertion that checks that the checkbox is enabled. Otherwise if you call this function twice it will enable it the first time and disable it the second time without throwing an error.